From b5491d304a17b53069005cc99fc6f20477c92506 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 1 Feb 2023 12:16:43 +0100
Subject: [PATCH] Fix build with gcc-13

* fixes:
com.webos.service.mediaindexer/1.0.0-16+gitAUTOINC+09b2b7d3b8-r0/git/src/perf/performancechecker.cpp:31:44:
error: 'name' has incomplete type
   31 | bool PerfChecker::addToPerfMap(std::string name)
      |                                ~~~~~~~~~~~~^~~~

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending
---
 src/perf/performancechecker.cpp                          | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/perf/performancechecker.cpp b/src/perf/performancechecker.cpp
index 4e270cb..16c88b7 100644
--- a/src/perf/performancechecker.cpp
+++ b/src/perf/performancechecker.cpp
@@ -16,6 +16,7 @@
 
 #include "performancechecker.h"
 #include "logging.h"
+#include <string>
 
 
 std::unique_ptr<PerfChecker> PerfChecker::instance_;
